src/components/TokenSelector.tsx

```diff
@@ -31,11 +31,11 @@ const Wrapper = styled.div`
 
   label {
     text-transform: uppercase;
-    padding-left: 0.5rem; // to align with Select input padding
+    padding-left: 0.5rem; /* to align with Select input padding */
   }
 
   input {
-    margin-left: 0; // to fix extra space on Select search box
+    margin-left: 0; /* to fix extra space on Select search box */
```

Comment:
That was me!
How did it work? Did it work?

Reply:
I assume it did, because styled-components' strips these comments anyway, but linter doesn't like them

Reply:
ahhhhh got it!